server/interaction/npc_test.go

```diff
@@ -0,0 +1,117 @@
+package interaction
+
+import (
+	"testing"
+
+	"github.com/joshprzybyszewski/cribbage/model"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_npcPlayer_buildDealAction(t *testing.T) {
```

Comment:
Maybe a good way to start writing tests would be to do

```go
func TestDumbNPC(t *testing.T) {
  // test all of the actions that an NPC needs to build in the testCases
}
```

Reply:
not sure that will work - each action needs to be tested differently (i.e. `DealCards` needs to make sure that the number of shuffles is within range, but `CribCards` needs to make sure the correct number of cards were thrown)